session/session.go

```diff
@@ -560,7 +572,8 @@ func (s *session) commitTxnWithTemporaryData(ctx context.Context, txn kv.Transac
 	sessVars := s.sessionVars
 	txnTempTables := sessVars.TxnCtx.TemporaryTables
 	if len(txnTempTables) == 0 {
-		return txn.Commit(ctx)
+		err := txn.Commit(ctx)
+		return err
```

Comment:
here is same as the `return txn.Commit(ctx)`

Reply:
Done